docs/components/webapp.rst

```diff
@@ -218,3 +218,42 @@ with stack traces by adding this to
       ('Your Name', 'your_email@domain.com'),
   )
   MANAGERS = ADMINS
+
+
+Running Web App in a Prod-like Way
+==================================
+
+By default, for local development when you run ``docker-compose up webapp`` it
+starts the web app geared towards local development. This means it starts
+Django's ``runserver`` and you most likely have ``DEBUG=True`` in your
+``my.env`` file. That means that static assets are automatically served from
```

Comment:
`DEBUG=True` comes from `docker/config/local_development.env`. So unless you're overriding that with your `my.env` file, then the `my.env` file doesn't really enter into it.

Reply:
Corrected now.